File: 01_Language/01_Functions/php/fputcsv.php
<?php

$list = array (
    array('aaa', 'bbb', 'ccc', 'dd,dd'),
    array('123', '456', '789'),
    array('"aaa"', '"bbb"')
);

$fp = fopen('test.csv', 'w');

foreach ($list as $fields) {
    fputcsv($fp, $fields);
}

fclose($fp);

File: 01_Language/01_Functions/python/fputcsv.py
# coding: utf-8

import csv

rows = [
    ["aaa", "bbb", "ccc", "dd,dd"],
    ["123", "456", "789"],
    ["\"aaa\"", "\"bbb\""],
]

with open("test.csv", mode="w") as f:
    writer = csv.writer(f, delimiter=",", quotechar='"')
    for row in rows:
        writer.writerow(row)
